tests/everflow/test_everflow_per_interface.py

```diff
@@ -213,3 +217,56 @@ def test_everflow_per_interface(ptfadapter, setup_info, apply_acl_rule, tbinfo,
     for port, ptf_idx in list(everflow_config['unselected_ports'].items()):
         logger.info("Verifying packet ingress from {} is not mirrored".format(port))
         send_and_verify_packet(ptfadapter, packet, exp_packet, ptf_idx, uplink_ports, False)
+
+
+def test_everflow_packet_format(ptfadapter, setup_info, apply_acl_rule, tbinfo,  # noqa F811
+                                toggle_all_simulator_ports_to_rand_selected_tor, ip_ver, erspan_ip_ver):  # noqa F811
+    """Verify that mirrored packets do not contain VLAN tags or unexpected fields."""
+    everflow_config = apply_acl_rule
+    packet, exp_packet = generate_testing_packet(ptfadapter, setup_info[UP_STREAM]['everflow_dut'],
+                                                 everflow_config['mirror_session_info'],
+                                                 setup_info[UP_STREAM]['ingress_router_mac'], setup_info, ip_ver,
+                                                 erspan_ip_ver)
+    uplink_ports = everflow_config["monitor_port_ptf_ids"]
+
+    # Send test packet
+    candidate_port, ptf_idx = list(everflow_config['candidate_ports'].items())[0]
+    logger.info(f"Sending test packet from candidate port {candidate_port}")
+    ptfadapter.dataplane.flush()
+    testutils.send(ptfadapter, pkt=packet, port_id=ptf_idx)
+
+    # Capture mirrored packet
+    logger.info("Capturing mirrored packet to verify format")
+    res = testutils.verify_packet_any_port(ptfadapter,
+                                           pkt=exp_packet,
+                                           ports=uplink_ports,
+                                           timeout=5)
+
+    # If res value is True, skip traffic test
+    if res:
```

Comment:
Hi @lerry-lee `verify_packet_any_port` returns a tuple or assert fails as shown https://github.com/p4lang/ptf/blob/main/src/ptf/testutils.py#L3430C20-L3437. But the `DummyTestUtils` returns a bool (True). The condition `if res:` will be considered true even when a valid index and packet is returned by the real verify call causing it to skip. Hence my earlier comment.